test/tests/functional/pbs_qsub_remove_files.py

```diff
@@ -103,26 +103,26 @@ def test_remove_files_error_file(self):
         submit a job with -Re option and make sure the error file
         gets deleted after job finishes and works with direct_write
         """
-        j = Job(TEST_USER, attrs={ATTR_k: 'de', ATTR_R: 'e'})
-        j.set_sleep_time(5)
         sub_dir = self.du.create_temp_dir(asuser=TEST_USER)
         mapping_dir = self.du.create_temp_dir(asuser=TEST_USER)
-        self.mom.add_config(
-            {'$usecp': self.server.hostname + ':' + sub_dir +
-             ' ' + mapping_dir})
-        self.mom.restart()
+        j = Job(TEST_USER, attrs={ATTR_k: 'de', ATTR_e: mapping_dir,
+                                  ATTR_R: 'e'})
+        j.set_sleep_time(5)
         jid = self.server.submit(j, submit_dir=sub_dir)
         self.server.expect(JOB, {ATTR_R: 'e'}, id=jid)
         self.server.expect(JOB, 'job_state', op=UNSET, id=jid)
-        for name in os.listdir(mapping_dir):
+        for name in os.listdir(sub_dir):
```

Comment:
@birbalsain Should not we check file is present or not in mapping directory?

Reply:
@visheshh stdout file should be present in submission directory because test direct write only stderr file in mapping directory and expect it will be deleted.